app/mailers/dossier_mailer.rb

```diff
@@ -98,4 +98,11 @@ def notify_groupe_instructeur_changed(instructeur, dossier)
 
     mail(from: NO_REPLY_EMAIL, to: instructeur.email, subject: @subject)
   end
+
+  def notify_dossier_not_submitted(dossier)
```

Comment:
Je pense qu'on peut appeler la méthode du mailer avec le même nom que la méthode métier.

```suggestion
  def notify_brouillon_not_submitted(dossier)
```